cve_bin_tool/checkers/liblas.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# pylint: disable=invalid-name
```

Comment:
What's pylint triggering on?  We don't really use pylint but should we be conforming to better naming standards here?

Reply:
I copied the structure from libjpeg.py and it had those, so I didn't removed this. Should I remove it and all the others?

I tested it now, It only returns a suggestion for "no class doc-string"

```
(venv39_CVE_Binary_tool) peb@ooo:cve-bin-tool$ pylint cve_bin_tool/checkers/liblas.py
************* Module cve_bin_tool.checkers.liblas
cve_bin_tool/checkers/liblas.py:14:0: C0115: Missing class docstring (missing-class-docstring)

-----------------------------------
Your code has been rated at 8.33/10

```

Reply:
Yeah, I think it's a leftover from a previous iteration of the checker files.   Let's remove those in another PR; I'll open an issue so we don't forget.  (You can do it or leave it as an easy bug for someone later; it's really not urgent)

Reply:
Filed issue #1247 so we can look at these later.